html/changelogs/Sindorman-cameras.yml

```diff
@@ -0,0 +1,12 @@
+author: PoZe
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - maptweak: "As per privacy update all bathrooms on the station no longer have cameras installed in them, with the exception of brig(prison) bathroom."
```

Comment:
...of the brig's bathroom. I'm pretty sure most know the prison is actually the brig, and aren't _exactly_  the same.

Reply:
There are two bathrooms in brig - one for security personally and one in the perma.